### PR TITLE
Document interface.macAddress

### PR DESCRIPTION
**Pull request by booxter**
This is to document https://github.com/kubevirt/kubevirt/pull/1213

### CODE REVIEW

**Comment by booxter** [comment]
@vladikr please review this update. This is the "MVP" change for the feature; a more elaborate rework of the documentation is in progress and will belong to another PR.

**Comment by fabiand** [comment]
To which PR is this related?

File: workloads/virtual-machines/interfaces-and-networks.md [hunk 1]
```diff
@@ -62,9 +62,10 @@ In some cases the underlying network plugin (flannel, weave, OpenShift SDN) acts
 
 ## Available interface attributes
 
-|Name|Possible values|Default|Description|
+|Name|Format|Default|Description|
 |--|--|--|--|
-|model|e1000, e1000e, ne2k_pci, pcnet, rtl8139, virtio|virtio|Interface model type exposed to guest (tip: use e1000 if your image doesn't support virtio)|
+|model|One of: e1000, e1000e, ne2k_pci, pcnet, rtl8139, virtio|virtio|Interface model type exposed to guest (tip: use e1000 if your image doesn't support virtio)|
+|macAddress|ff:ff:ff:ff:ff:ff or FF-FF-FF-FF-FF-FF||MAC address as seen inside the guest system|
```
Comment (fabiand) [inline workloads/virtual-machines/interfaces-and-networks.md]: Please add an example.

Comment (booxter) [reply workloads/virtual-machines/interfaces-and-networks.md]: Not sure if that's what you mean, but I added one in description. The format column already gave enough context though so not sure if that's what you were asking.

Comment (vladikr) [reply workloads/virtual-machines/interfaces-and-networks.md]: I think @fabiand is asking to update the yaml example on line 31. Perhaps we had to update it with the introduction of the `model` attribute.

Comment (vladikr) [reply workloads/virtual-machines/interfaces-and-networks.md]: Looks good to me, besides the yaml exmaple update.

Comment (booxter) [reply workloads/virtual-machines/interfaces-and-networks.md]: OK gotcha; I've added macAddress (and model) there. I haven't touched e.g. delegateIP that doesn't really exist for now. This is belong to a different PR.

**Pull request merged**